specificatie/persoon.yaml

```diff
@@ -0,0 +1,216 @@
+openapi: 3.0.3
+info:
+  title: Persoon definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    GemeenteVanInschrijving:
+      description: |
+        Een code die aangeeft in welke gemeente de persoon woont, of de laatste gemeente waar de persoon heeft gewoond, of de gemeente waar de persoon voor het eerst is ingeschreven.
+      type: string
+      pattern: ^[0-9]{4}$
+      example: "0518"
+    GbaPersoonBeperkt:
```

Comment:
waarom is deze opgenomen in de historie API? die wordt hier niet gebruikt

Reply:
Is verwijderd.

Reply:
kan je een nog wat bredere scan doen van overbodige componenten, want die zijn er nog wel. Bijvoorbeeld IndicatieCurateleRegister, Leeftijd, Reisdocumentnummer, PersoonBeperkt, enz.